libs/ui-components/src/components/form/FormSelect.tsx

```diff
@@ -72,33 +63,22 @@ const FormSelect: React.FC<FormSelectProps> = ({ name, items, helperText, placeh
           setIsOpen(open);
         }}
       >
-        {itemKeys.length && (
+        {itemKeys.length ? (
```

Comment:
this would be better
```suggestion
        {!!itemKeys.length &&  (
```

Reply:
That also works, thanks!